originals/b.yaml

```
@@ -302,3 +314,10 @@
     theme:
     - Management
 
+- name: BVE Trainsim
+  names:
+  - Boso View Express
+  external:
+    wikipedia: BVE Trainsim
+  platform:
+  - Windows
```

```suggestion
  - Windows
  meta:
    genre:
    - Simulation
    subgenre:
    - Train Simulator
```
Add "Train Simulator" to subgenres schema